.github/workflows/pr-checks-for-qa-approvals.yml

```diff
@@ -0,0 +1,70 @@
+
+#
+# Checks if QA approvals are necessary, and if so runs the associated action when a new review is added
+#
+
+name: '[Utils] QA required approval'
+
+on:
+  pull_request:
+    types: [opened, edited, converted_to_draft, ready_for_review, reopened]
+    paths:
+      - 'VAMobile/src/**'
+      - 'VAMobile/package.json'
+      - '!VAMobile/src/**.test.*'
+  pull_request_review:
+    types: [submitted, dismissed]
+  pull_request_target:
+    paths:
+      - 'VAMobile/src/**'
+      - 'VAMobile/package.json'
+      - '!VAMobile/src/**.test.*'
+    branches:
+      - 'develop'
+      
+jobs:
+  check_for_qa_approval:
```

Comment:
I realized this workflow will make QA's approval required for Dependabot PRs also, which those don't typically need QA approval to merge. Can we add logic to this job that makes it return successfully if the PR was opened by Dependabot? 

The easiest way to do this is probably to check for the author of the PR. This should be accessible via the `github` context:

```yml
github.event.pull_request_review.user.login == 'dependabot[bot]'
```

Reply:
@theodur please review whenever you can